File: app/src/main/java/com/softhink/single/login/LoginActivity.java
package com.softhink.single.login;

import android.os.Bundle;
import com.softhink.single.BaseActivity;
import com.softhink.single.Constants;
import com.softhink.single.R;

public class LoginActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

        mFragmentManager.beginTransaction()
                .add(R.id.containerLogin, new LoginFragment(),
                Constants.INSTANCE.getLOGINFRAGMENT())
                .commit();
    }

}

File: app/src/main/java/com/softhink/single/login/ui/LoginActivity.kt
package com.softhink.single.login.ui

import android.os.Bundle
import com.softhink.single.BaseActivity
import com.softhink.single.Constants
import com.softhink.single.R

class LoginActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        mFragmentManager.beginTransaction()
                .add(R.id.containerLogin, LoginFragment(),
                        Constants.LOGINFRAGMENT)
                .commit()
    }

}
